src/components/ui/QuantitySelector/quantity-selector.scss

```diff
@@ -0,0 +1,53 @@
+[data-store-quantity-selector] {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  width: 8.125rem;
+  height: var(--space-7);
+  background-color: var(--bg-neutral-lightest);
+  border: var(--border-width-0) solid var(--color-border-input);
+  border-radius: var(--border-radius-default);
+
+  [data-store-icon] {
+    margin: 0;
+    color: var(--color-border-input-selected);
+    line-height: 0;
+  }
+
+  [data-quantity-selector-input] {
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    text-align: center;
+    background-color: transparent;
+    border-width: 0;
+
+    &:focus { outline: 0; }
+  }
+
+  [data-quantity-selector-button] {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    background-color: transparent;
+    border-color: transparent;
+
+    &:disabled {
+      cursor: not-allowed;
+      [data-store-icon] { color: var(--color-border-input-disabled); }
+    }
+  }
+
+  &[data-store-quantity-selector="disabled"] {
+    background-color: var(--bg-disabled);
+    border-color: var(--bg-disabled);
+  }
+
+  &:hover:not([data-store-quantity-selector="disabled"]) {
+    border: var(--border-width-2) solid var(--color-border-input-selected);
+  }
```

Comment:
It looks like `focus` and `hover` states' have different styles. You can play a little getting the [SkuSelector](https://github.com/vtex-sites/base.store/blob/main-styled/src/components/ui/SkuSelector/sku-selector.scss#L37-L47) as a reference.